readme.md

```diff
@@ -232,6 +232,69 @@ import kafka.serializer.DefaultDecoder
 }
 ```
 
+
+AkkaHighLevelConsumer
```

Comment:
@coreyauger I am thinking we should choose a name other than HighLevel for this consumer, because the other consumers also do use the HighLevelConsumer under the hood.

Maybe something like `AkkaDirectConsumer` ?